app/uk/gov/hmrc/merchandiseinbaggage/service/CustomsDutyCalculator.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ */
+
+package uk.gov.hmrc.merchandiseinbaggage.service
+
+import java.time.LocalDate
+
+import cats.data.EitherT
+import cats.instances.future._
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+import uk.gov.hmrc.merchandiseinbaggage.connectors.CurrencyConversionConnector
+import uk.gov.hmrc.merchandiseinbaggage.model.api.{CalculationRequest, CurrencyConversionResponse}
+import uk.gov.hmrc.merchandiseinbaggage.model.core.AmountInPence._
+import uk.gov.hmrc.merchandiseinbaggage.model.core.{AmountInPence, BusinessError, CurrencyNotFound}
+import uk.gov.hmrc.merchandiseinbaggage.repositories.CustomsRate._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+trait CustomsDutyCalculator extends CurrencyConversionConnector {
+
+  def customDuty(httpClient: HttpClient, calculationRequest: CalculationRequest)
+                (implicit hc: HeaderCarrier, ec: ExecutionContext): EitherT[Future, BusinessError, AmountInPence] =
+    for {
+      lt         <- EitherT.liftF(findCurrencyConversion(httpClient, calculationRequest.currency, LocalDate.now))
```

Comment:
maybe rename 'lt' to something like 'rates' or 'currencyConversionRates'

Reply:
done